packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -218,7 +222,16 @@ const AppModals = observer(() => {
         ComponentToLoad = <WalletsUpgradeLogoutModal />;
     }
 
-    if (should_show_effortless_login_modal && !is_tour_open && !is_from_derivgo && !is_onboarding) {
+    if (
```

Comment:
`suggestion`

What I am raising is not a blocker but maybe it will increase more readability 

`const showEffortlessLoginModal = should_show_effortless_login_modal &&
    !is_tour_open &&
    !is_from_derivgo &&
    !is_onboarding &&
    !should_show_appropriateness_warning_modal &&
    !should_show_risk_warning_modal &&
    !is_from_signup_account &&
    !is_real_acc_signup_on;

if (showEffortlessLoginModal) {
    ComponentToLoad = <EffortlessLoginModal />;
}`

Assign the complex condition to a descriptive variable, and then use that variable in the if statement. This keeps the logic clear and concise. 😬